internal/services/config/config.go

```diff
@@ -134,6 +134,8 @@ type Notification struct {
 
 	WebhookURL    string `yaml:"webhookURL"`
 	WebhookSecret string `yaml:"webhookSecret"`
+
+	RunWebhookExpireInterval time.Duration `yaml:"runWebhookExpireInterval"`
```

Comment:
With this single option you don't have a way disable older webhook deletion.
Is this ok or there was also the idea to make this optional?

If not than the commit title and description are misleading. They should be `notification: delete old run webhooks` and `this patch implements automatic deletion of run webhooks older than the defined interval with a default of 7 days`.

Reply:
So let's keep it as is and fix the commit title/description as suggested.